lib/improver/nowcasting/accumulation.py

```diff
@@ -221,17 +243,7 @@ def _get_cube_subsets(self, cubes, forecast_period, integral):
         constr = iris.Constraint(
             forecast_period=lambda fp: start_point <= fp <= forecast_period)
 
-        cube_subset = cubes.extract(constr)
-
-        if len(cube_subset) != int(integral + 1):
-            msg = (
-                "The provided cubes result in a partial period given the "
-                "specified accumulation_period, i.e. the number of cubes "
-                "is insufficient to give a set of complete periods. Only "
-                "complete periods will be returned.")
-            warnings.warn(msg)
-            return None
-        return cube_subset
+        return cubes.extract(constr)
```

Comment:
Can you please remove the warns section from the doc-string having removed this warning.,

Reply:
Removed warning.